vc/vcjwt.go

```diff
@@ -0,0 +1,146 @@
+package vc
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"slices"
+	"time"
+
+	"github.com/tbd54566975/web5-go/jwt"
+)
+
+// Verify verifies the decoded vc-jwt. It checks for the presence of required fields and verifies the jwt.
+// It returns the decoded vc-jwt and an error if the verification fails.
```

Comment:
```suggestion
// It returns the decoded vc-jwt and the verification result.
```

Technically the decoding could error-out prior to the call to verify